README.md

```diff
@@ -27,10 +27,14 @@ Three primary elements make up the CONNECT solution:
 
 History
 -------
+* 4.4 released December 2014
+* 4.3 released March 2014
+* 4.2 released
+* 4.1 released
```

Comment:
Do we not have dates for these two releases?  (or does it make more sense to remove this information and use the wiki + HISTORY.md for this?)